courses/free-courses-en.md

```diff
@@ -1111,6 +1113,7 @@
 * [Frontend UI Development with React](https://www.youtube.com/playlist?list=PL0vfts4VzfNgUUEtEjxDVfh4iocVR3qIb) - Jeff Delaney, Fireship.io (YouTube)
 * [Full Stack Foundations](https://www.udacity.com/course/full-stack-foundations--ud088) - by AWS on Udacity
 * [Full stack open](https://fullstackopen.com/en/) - University of Helsinki, Houston Inc., Terveystalo, Elisa, K-ryhmä, Unity Technologies, Konecranes
+* [Full Stack Web Developement Bootcamp](https://www.appacademy.io/course/app-academy-open) - App Academy
```

Comment:
link does not go to the course. Can't use it here

Reply:
![image](https://user-images.githubusercontent.com/103281314/194136701-e5df4c50-8549-4c82-bef1-66f1524b8c27.png)
@eshellman by scrolling down a bit you can get all the infos you will gain there. and also you need to signup if you want to take the bootcamp.

Reply:
not good enough. remove it

Reply:
@eshellman should i remove the third one also?